packages/nx/src/utils/async-iterator.ts

```diff
@@ -0,0 +1,16 @@
+export function isAsyncIterator<T>(v: any): v is AsyncIterableIterator<T> {
+  return typeof (v as any)?.[Symbol.asyncIterator] === 'function';
```

Comment:
Don't need the `(v as any)` since it's already an any type. Might consider `v: unknown` so `v` is type-safe within the function body (in this case it's one-line so doesn't matter too much).